src/program/services/post_processing/media_analysis.py

```
@@ -139,23 +139,19 @@ def _analyze_with_ffprobe(self, file_path: str, item: MediaItem) -> Optional[Dic
+                ffprobe_dict = media_metadata.model_dump_json()
                 logger.debug(f"FFprobe analysis successful for {item.log_string}")
                 return ffprobe_dict
-            else:
-                logger.warning(f"FFprobe returned no data for {item.log_string}")
-                return None
 
-        except Exception as e:
-            logger.error(f"FFprobe analysis failed for {item.log_string}: {e}")
-            return None
+            logger.warning(f"FFprobe returned no data for {item.log_string}")
+            return {}
```

Ruff (0.13.3)

Consider moving this statement to an `else` block

(TRY300)